python/README.rst

```diff
@@ -0,0 +1,18 @@
+=====
+pyddq
+=====
+
+Python binding to Drunken Data Quality
+
+
+Description
+===========
+
+A LONGER DESCRIPTION OF YOUR PROJECT GOES HERE...
```

Comment:
Yes, it makes sense to fill in this description since this README will be available at PyPi when we upload the package.